README.md

```diff
@@ -64,6 +67,42 @@ The simdjson library is easily consumable with a single .h and .cpp file.
    100 results.
    ```
 
+On Demand Alpha: Quick Start
```

Comment:
It is fine and even wise to advertise the "alpha" nature of the work, but I would do so just at the beginning of the document rather than to pollute the header with the word 'alpha'.

Reply:
Fixed.